src/main/kotlin/gg/essential/elementa/UIComponent.kt

```diff
@@ -351,7 +367,15 @@ abstract class UIComponent : Observable() {
     }
 
     protected fun getMousePosition(): Pair<Float, Float> {
-        return UIComponent.getMouseX() to UIComponent.getMouseY()
+        return transformMouseLocation(getMouseX(), getMouseY())
+    }
+
+    internal fun transformMouseLocation(mouseX: Float, mouseY: Float): Pair<Float, Float> {
```

Comment:
`transform` doesn't really say much about what this actually does, just that it does *something*. Something like `pixelCoordinatesToPixelCenter` would be way better imo.